src/Compilers/CSharp/Test/Semantic/Semantics/TargetTypedDefaultTests.cs

```diff
@@ -988,7 +1064,7 @@ static void Main()
             var nodes = tree.GetCompilationUnitRoot().DescendantNodes();
 
             var def = nodes.OfType<DefaultLiteralSyntax>().Single();
-            Assert.Null(model.GetTypeInfo(def).Type);
+            Assert.Equal("System.Int16", model.GetTypeInfo(def).Type.ToTestDisplayString());
             Assert.Null(model.GetSymbolInfo(def).Symbol);
             Assert.Null(model.GetDeclaredSymbol(def));
```

Comment:
>Assert.Null(model.GetDeclaredSymbol(def)); [](start = 12, length = 42)

It would be good to test conversion and constant value for this scenario because it is using explicit cast.